resources/lang/de/command/messages.php

```diff
@@ -5,80 +5,80 @@
+        'created' => 'Neue Location (:name) mit der ID :id erstellt.',
         'deleted' => 'Location gelöscht.',
     ],
     'user' => [
         'search_users' => 'Gebe einen Nutzernamen, eine UUID oder eine Email an',
-        'select_search_user' => 'ID des Users (Enter \'0\' to re-search)',
+        'select_search_user' => 'ID des Users (Gebe \'0\' ein zum Neu suchen)',
         'deleted' => 'Benutzer erfolgreich gelöscht.',
         'confirm_delete' => 'Bist du dir wirklich sicher?',
         'no_users_found' => 'Es wurden keine User gefunden.',
         'multiple_found' => 'Es wurden mehrere Accounts gefunden.',
-        'ask_admin' => 'Is this user an administrator?',
+        'ask_admin' => 'Ist dieser Benutzer ein Administrator?',
         'ask_email' => 'Email Adresse',
-        'ask_username' => 'Username',
-        'ask_name_first' => 'Vornamee',
+        'ask_username' => 'Benutzername',
+        'ask_name_first' => 'Vorname',
         'ask_name_last' => 'Nachname',
-        'ask_password' => 'Password',
-        'ask_password_tip' => 'Wenn du dass wirklich tun willst drücke Strg+c und benutze das `--no-password` flag.',
+        'ask_password' => 'Passwort',
+        'ask_password_tip' => 'Wenn du das wirklich tun willst, drücke Strg+C und benutze das `--no-password` Flag.',
         'ask_password_help' => 'Das Passwort muss Zahlen, Groß- und Kleinbuchstaben enthalten und mindestens 8 Zeichen lang sein.',
         '2fa_help_text' => [
-            'This command will disable 2-factor authentication for a user\'s account if it is enabled. This should only be used as an account recovery command if the user is locked out of their account.',
-            'If this is not what you wanted to do, press CTRL+C to exit this process.',
+            'Dieser Befehl deaktiviert die 2-Faktor-Authentifizierung für das Konto eines Benutzers, wenn sie aktiviert ist. Dies sollte nur dann als Befehl zur Wiederherstellung eines Kontos verwendet werden, wenn der Benutzer von seinem Konto ausgeschlossen ist.',
+            'Wenn das nicht das ist was du tun wolltest, drücke Strg+C um den Prozess zu beenden.',
         ],
-        '2fa_disabled' => '2-Factor authentication wurde für :email deaktivier.',
+        '2fa_disabled' => '2-Factor Authentifizierung wurde für :email deaktiviert.',
     ],
     'schedule' => [
-        'output_line' => 'Dispatching job for first task in `:schedule` (:hash).',
+        'output_line' => 'Einplanungsjob für die erste Aufgabe in `:schedule` (:hash).',
     ],
     'maintenance' => [
-        'deleting_service_backup' => 'Deleting service backup file :file.',
+        'deleting_service_backup' => 'Lösche Service Backup Datei :file.',
     ],
     'server' => [
-        'rebuild_failed' => 'Rebuild request for ":name" (#:id) on node ":node" failed with error: :message',
+        'rebuild_failed' => 'Neuaufbauanforderung für ":name" (#:id) auf Node ":node" ist gescheitert mit Fehler: :message',
```

Comment:
"Neubau" is better than "Neuaufbau" and also add a "dem" in between "mit" and "Fehler".